db/Controllers/news.controllers.js

```diff
@@ -10,4 +10,14 @@ const getTopics = (request, response, next) => {
     });
 };
 
-module.exports = { getTopics };
+const getApi = (request, response, next) => {
+  selectApi()
```

Comment:
There isn't a need for a model function for this, model functions are for interacting with the database